app/_components/inputs/commodities/commodities.tsx

```diff
@@ -19,10 +21,12 @@ const CommoditiesField: React.FC<CommodityProps> = ({ control }) => {
   // TODO: for now these values have leading designators that we need to remove
   // for example: 'water' is stored as 'ch_water' designated as a chemical
   // if we don't end up using this designation then we should remove it
+
+  // TODO: while commodity names are now capitalised, names which contain acronyms such as CMM Composites are not capitalised correctly
   const formattedCommodities: CommodityGroup[] = commodities.map(
     (commodity) => ({
       value: commodity.slice(3).split('_').join(' '),
-      label: commodity.slice(3).split('_').join(' '),
+      label: titleCase(commodity.slice(3).split('_').join(' ')),
```

Comment:
For the moment, I might just leave them lower case since AFAIK we aren't sure whether this is being stored locally or pulled from the backend. We could even put it in a monospace font or something to make it look like it's intended...